x-pack/test/saved_object_api_integration/common/fixtures/namespace_agnostic_type_plugin/server/index.ts

```diff
@@ -1,64 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-import { CoreSetup, Plugin } from 'src/core/server';
-import { PluginSetupContract as FeaturesPluginSetupContract } from '../../../../../../plugins/features/server';
-
-export const plugin = () => new NamespaceAgnosticTypePlugin();
-
-interface SetupDeps {
-  features: FeaturesPluginSetupContract;
-}
-
-class NamespaceAgnosticTypePlugin implements Plugin {
-  setup(core: CoreSetup, plugins: SetupDeps) {
-    core.savedObjects.registerType({
-      name: 'globaltype',
-      hidden: false,
-      namespaceType: 'agnostic',
-      management: {
-        importableAndExportable: true,
-      },
-      mappings: {
-        properties: {
-          title: {
-            type: 'text',
-            fields: {
-              keyword: {
-                type: 'keyword',
-                ignore_above: 2048,
-              },
-            },
-          },
-        },
-      },
-    });
-
-    plugins.features.registerFeature({
```

Comment:
__question:__ I see we don't do this in a new plugin, we don't need this anymore?

Reply:
Good catch. No, we don't need it for saved object API integration tests.

I looked at the history and it was originally added during #31652, which was a huge PR. My guess is it was a WIP for feature integration tests that were never completed, or it was leftover cruft that was copied from another test plugin, or there were feature integration tests here that got refactored out later on. In any case, we should be safe to remove it here (CI will confirm!)